### ISSUE TITLE
search problem

### ISSUE THREAD

**Issue by aeqvkec** [sabrogden/Ditto#240]
I have a group named "7A" and within "7A" are more groups like "words", "phrases", "texts".
I also have a clip named "project" in group "words".
Now I click inside group "7A" and I can see something like this.

<7A
words
phrases
texts

Now I search for a clip like "project" in all the clips in "7A".
It just doesn't work. only shows     There's no result for "project".

How can this be fixed? It would be very useful. Many thanks!

**Comment by sabrogden** [comment]
Yes i see what you are saying.  Ditto does not search child groups.  I'll try and look at how to do this with sqlite.

**Comment by Chealer** [comment]
I confirm this in 03.25.113, but this does not occur in the root/top group (in other words, in "No Group").

Thank you for reporting @aeqvkec, but this ticket's title is very vague (which probably resulted in someone reporting the same issue in ticket #717). Please give it a more specific title. For example:

> Search in a group excludes clips from its descending groups

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
